settings/src/components/first-time/first-time.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useContext, useRef } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import { lock, edit, reusableBlock } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import ScreenNavigation from '../screen-navigation';
+import TOTP from '../totp';
+import WebAuthn from '../webauthn/webauthn';
+import BackupCodes from '../backup-codes';
+import SetupProgressBar from './setup-progress-bar';
+import Home from './home';
+import WordPressLogo from './wordpress-logo';
+import { GlobalContext } from '../../script';
+
+/**
+ * Check if the URL is valid. Make sure it stays on wordpress.org.
+ *
+ * @param  url
+ * @return {boolean} Whether it's a valid URL.
+ */
+const isValidUrl = ( url ) => {
+	try {
+		const { hostname } = new URL( url );
+		return hostname.endsWith( 'wordpress.org' );
+	} catch ( exception ) {
+		return false;
+	}
+};
+
+/**
+ * Render the correct component based on the URL.
+ */
+export default function FirstTime() {
+	const modalRef = useRef( null );
+	const { navigateToScreen, screen } = useContext( GlobalContext );
+	const steps = [
+		{
+			title: 'Choose an authentication method',
```

Comment:
Do these strings need to be translatable?

Reply:
Not available for this project yet. 

It's logged: https://github.com/WordPress/wporg-two-factor/issues/273